main/webapp/modules/core/langs/translation-en.json

```
@@ -548,6 +548,8 @@
     "core-views/judg-hist": "Judgment action timestamp",
     "core-views/hist-entries": "$1: judgment action timestamp",
     "core-views/actions": "Actions",
+    "core-views/add-column-with-URLs-of-matched-entities":"Add column with URLs of matched entities",
```

To indicate that there is a dialog showing up after one clicks this entry, we have the convention of adding "…" in English at the end.
```suggestion
    "core-views/add-column-with-URLs-of-matched-entities":"Add column with URLs of matched entities…",
```